hotsos/core/plugins/storage/ceph.py

```diff
@@ -91,6 +91,14 @@ def csv_to_set_inner(*args, **kwargs):
     return csv_to_set_inner
 
 
+class CephTimestampMatcher(TimestampMatcherBase):
+
+    @property
+    def patterns(self):
+        return [r'^(?P<year>\d{4})-(?P<month>\d{2})-(?P<day>\d{2})\s?T'
```

Comment:
Ceph logs include the milliseconds too. E.g. `2023-04-18T00:00:51.995+0000 7fdd55879700  4 rocksdb:`. But I think this pattern should work in any case.

IIRC, older log formato allowed `' '` (space) in place of `T`. So you need to change to cover either: `[ |T]`.

Reply:
ack have fixed